src/builders/AvailabilityBuilder.ts

```diff
@@ -1,5 +1,13 @@
+import {
+  defaultPricing,
```

Comment:
keep just types in the types package, remove this default pricing, it should be just here in octomock @michalpansky